docs/releases/release-8-3.md

```diff
@@ -0,0 +1,417 @@
+
+Key improvements made to the installer include:
+- `memtest86+` has been updated to version 6 with EFI support.
+- The `xen-pciback.hide` boot parameter (useful for PCI passthrough) will now persist when upgrading XCP-ng (merged upstream).
+- The default SR type now defaults to `ext`, and users are presented with radio buttons instead of a checkbox for the choice (not contributed upstream as it’s irrelevant for XenServer).
+- The "Do you want to install any supplemental packs?" dialog was removed at the end of installation (submitted upstream to make it optional, so we can disable it while maintaining upstream consistency). Exception: it still appears if a driver disk was loaded during installation.
+- The primary disk selection screen always appears, even if only one disk is available. Previously, if only one disk met the prerequisites, the screen was skipped, leading users to mistake the Local SR creation screen for the primary disk selection (merged upstream).
+- Better error message granularity when installations fail due to incorrect system dates, signature issues, etc. (merged upstream).
+- Improved robustness when installing on used disks that contain ZFS metadata.
+- Earlier warnings if attempting to upgrade a system with a different BIOS type (Legacy BIOS vs. UEFI), preventing issues later in the process.
+- The installer now blocks upgrades if the host's certificate key length is less than 2048 bits. See [Host certificate key too small prevents upgrade](#host-certificate-key-too-small-prevents-upgrade).
+- IPv6 support was added. Although not accepted upstream yet, it was important enough to retain in XCP-ng's installer version, with hopes that XenServer developers will review it in the future.
+- Various bugfixes.
+
+Our contribution was not (yet?) accepted for:
+- LACP bond creation from the installer.
+- Improved handling of existing software RAIDs.
+
+### VM anti-affinity [XS]
+
+XenServer added a new feature which allows to constitute anti-affinity VM groups, meaning that you would like to avoid VMs to be started, or evacuated from another host, towards a host where another VM of the same group already runs.
+
+For more information, refer to [XenServer's documentation](https://docs.xenserver.com/en-us/xenserver/8/vms/placement).
+
+### PCI passthrough support in the API
+
+We expanded the XAPI API by adding endpoints to manage PCI passthrough devices. Xen Orchestra developers added corresponding UI elements, enabling users to add or remove devices from the passthrough list. Once a reboot of the host, these devices are released by dom0 (the control domain) and become available for assignment to VMs.
```

Comment:
`Once a reboot of the host` +has been done, maybe ? Or once the host has rebooted.

Reply:
Fixed